custom/icds_reports/models.py

```diff
@@ -1060,3 +1066,51 @@ class Meta:
         app_label = 'icds_model'
         managed = False
         db_table = 'ucr_table_name_mapping'
+
+
+@architect.install(
+    'partition',
+    type='range',
+    subtype='date',
+    constraint='month',
+    column='time_of_use_start',
+    db=get_icds_ucr_db_alias()
+)
+class ICDSAuditEntryRecord(models.Model):
+    username = models.EmailField(db_index=True)
+    assigned_location = models.CharField(max_length=256, null=True)
```

Comment:
It looks like this could be multiple location ids. Can you change it to `assigned_location_ids` and to an `ArrayField` https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#arrayfield